include/utils/perf_log.h

```diff
@@ -249,6 +249,11 @@ class PerfLog
    */
   PerfData get_perf_data(const std::string & label, const std::string & header="");
 
+  /**
+   * \returns the raw underlying data structure for the entire performance log.
+   */
+  const std::map < std::pair<std::string, std::string>, PerfData > & get_log_raw() const { return log; };
```

Comment:
No semi-colon is needed after the closing curly brace. Clang warns about this [-Wextra-semi] for every file that includes perf_log.h